networkServer/src/main/java/org/pesc/cds/web/DocumentController.java

```diff
@@ -330,26 +348,40 @@ public Transaction sendFile(
             @RequestParam(value="file_format", required=true) String fileFormat,
             @RequestParam(value="document_type", required=false) String documentType,
             @RequestParam(value="department", required=false) String department,
-            @RequestParam(value="school_code", required=true) String schoolCode,
-            @RequestParam(value="school_code_type", required=true) String schoolCodeType
+            @RequestParam(value="source_school_code", required=false) String sourceSchoolCode,
+            @RequestParam(value="source_school_code_type", required=false) String sourceSchoolCodeType,
+            @RequestParam(value="destination_school_code", required=true) String destinationSchoolCode,
+            @RequestParam(value="destination_school_code_type", required=true) String destinationSchoolCodeType,
+            @RequestParam(value="student_release", required=false) Boolean trStudentRelease,//required=true
```

Comment:
Some of the comments here conflict with the actual required value. This can cause confusion.